docs/recipes/ci-configurations/circleci-workflows.md

```diff
@@ -22,14 +22,12 @@ Finally, we call our release job with a `requires` parameter so that `release` w
 ```yaml
 version: 2.1
 orbs:
-  node: circleci/node@4.5
+  node: circleci/node@5.0.0
 jobs:
   release:
     executor: node/default
     steps:
       - checkout
-      - node/install
```

Comment:
the reason we include this step is to encourage using the latest lts version of node when releasing, even if that is not the version used to verify the project. does removing this step still accomplish that goal? if not would including a comment make that goal more clear here?

Reply:
Thanks for replying!

The reason why that step can be removed is, `circleci/node`'s default execute already use lts node docker image.

(There is a configurable tag, which defaults to 'lts')


Reference: https://circleci.com/developer/orbs/orb/circleci/node#executors-default